addons/jam/magwells_762x51.hpp

```diff
@@ -35,6 +35,10 @@
     class CBA_762x51_M14_XL {};     // M14 drum mag (X-Products 50 round)
 
     class CBA_762x51_MG3 {};        // MG3 DM1 link belts
+    class CBA_762x51_MkI_EMR {      // Mk-I EMR 7.62 mm magazines
+        BI_mags[] = {
+            "20Rnd_762x51_Mag"
+        };
```

Comment:
missing closing };

Reply:
Thanks, that's what I get for copy/pasting